docs/Home/index.md

```diff
@@ -3,17 +3,17 @@
 
 ## What is Clinica ?
 Clinica is a software platform for clinical neuroscience research studies using multimodal data and most often longitudinal follow-up.
-You can learn more on [this page](WhatIsClinica).
+You can learn more on [this page](../WhatIsClinica.md).
 
 ## Installation
 
 Clinica can be installed on **MacOS** and **Linux** (CentOS or Debian/Ubuntu) machines,
 and possibly on Windows computers with a Linux Virtual Machine.
 We assume that users installing and using Clinica are comfortable using the command line.
 
-- [Installation](Software/Installation.md)
-- [Third-party software](Software/Third-party.md)  
-- [Interacting with Clinica](Software/InteractingWithClinica.md)
+- [Installation](../Software/Installation.md)
+- [Third-party software](../Software/Third-party.md)  
+- [Interacting with Clinica](../Software/InteractingWithClinica.md)
```

Comment:
Is it on purpose that you didn't add a bullet point for "contributing to Clinica" ?

Reply:
No, I forgot !